service-myhpi/src/main/kotlin/de/hpi/cloud/myhpi/MyHpiService.kt

```diff
@@ -143,7 +146,7 @@ class MyHpiServiceImpl(private val bucket: Bucket) : MyHpiServiceGrpc.MyHpiServi
         return Action.newBuilder().buildWithDocument(this) {
             id = getString(KEY_ID)
             title = it.getI18nString("title")
-            it.getString("icon")?.let { i -> icon = i }
+            it.getString("icon")?.let { i -> icon = ByteString.copyFrom(i.decodeBase64()) }
```

Comment:
Is there a reason to specifically use ByteString?

Reply:
ByteString is the Java representation for a field of type `bytes`